ee/models/assistant.py

```diff
@@ -56,7 +64,7 @@ class ConversationCheckpointBlob(UUIDModel):
     class Meta:
         constraints = [
             models.UniqueConstraint(
-                fields=["checkpoint_id", "channel", "version"],
+                fields=["thread_id", "checkpoint_ns", "channel", "version"],
```

Comment:
This will lock the table anyway, so I didn't set checkpoint_ns to null. The table was just created, so a short lock shouldn't be a problem.